common/discordutil.js

```diff
@@ -72,5 +86,27 @@ module.exports = {
         delete file.ids[channelID];
         this.writeToFile();
         return true;
+    },
+    createPreviewMessage(message, text, images) {
+        const authorName = `${message.author.username}#${message.author.discriminator} (ID ${message.author.id})`
+        var pictureLinks = ''
+        if (images.size){
+            images.forEach(image => {
+                pictureLinks += `${image['proxyURL']}\n` ;
+            })
+        }
+        console.log(pictureLinks);
```

Comment:
should we log this? and if so, maybe add a 'Creating preview message with links...' or something so its easy to locate in logs?